src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/anonymization/SpecialAnonymizationUserPage.java

```diff
@@ -1,8 +1,10 @@
 package com.wikia.webdriver.pageobjectsfactory.pageobject.special.anonymization;
 
 import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.core.url.UrlBuilder;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialPageObject;
 
+import com.sun.javafx.fxml.builder.URLBuilder;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this usage of Sun classes by ones from the Java API. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1191)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.sun.javafx.fxml.builder.URLBuilder'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)